tests_integration/test_main.py

```diff
@@ -542,3 +545,73 @@ def test_protocols_udp_restricted_port_values_start_must_be_lte_end(
 
         stderr = ex.value.stderr.decode()
         assert "Start port value must be less or equal to end port value" in stderr
+
+
+@pytest.mark.integration()
+def test_resource_access_flows(kopf_settings, unique_resource_name):
+    assert "TWINGATE_PRINCIPAL_ID" in os.environ
```

Comment:
Lets rename `TWINGATE_PRINCIPAL_ID` to `TWINGATE_TEST_PRINCIPAL_ID` so we know this argument is used for tests

Reply:
I added a valid `TWINGATE_TEST_PRINCIPAL_ID` to CI so your PR can pass tests

Reply:
recommended change applied